src/llmcompressor/modifiers/awq/base.py

```diff
@@ -677,6 +660,19 @@ def _assert_all_activations_consumed(self):
             raise RuntimeError("Some cached activations were not used")
 
 
+@torch.no_grad()
+@torch.compile()
+def _compute_loss(
+    fp16_output: torch.Tensor,
+    int_w_output: torch.Tensor,
+) -> torch.Tensor:
+    """
+    Compute MSE loss over the flattened output of all batches
+    """
+    return (fp16_output - int_w_output).view(-1).float().pow(2).mean()
+
+
+@torch.compile()
```

Comment:
I wonder if torch.compile performs better without the asserts?

Reply:
Tracing has asserts turned off by default
https://github.com/pytorch/pytorch/blob/main/torch/fx/proxy.py#L131

I wasn't sure about torch.compile